docs/administration/managing-infrastructure/tentacle-configuration-and-file-storage/manually-uninstall-tentacle.md

```diff
@@ -47,7 +47,11 @@ These steps will remove all traces of Octopus Tentacle from your computer:
 :::problem
 This will also remove your deployed applications if you have not configured Tentacle to use a different Application folder. See below for more details.
 :::
-4. Find and delete the Octopus registry entries from **`HKLM\SOFTWARE\Octopus`**.
+4. Find and delete the Octopus registry entries from the following locations:
+    * **`HKLM\SOFTWARE\Octopus`** 
+    * **`HKLM\SOFTWARE\Classes\Installer\Products`**
```

Comment:
It may be good to explain how to determine which of the entries under these sections are to be deleted, to make sure they don't delete the entire (say) Products keys - especially given that the first line specifies the key to delete, while the last three have data inside them that needs to be deleted.  Does that make sense?

Reply:
I'd agree with @MissedTheMark here. Looking at a local Octopus Server install which also has a Tentacle installed on it, I'd be wary of deleting `HKLM\SOFTWARE\Octopus` since it also contains the Octopus Server product:

![image](https://user-images.githubusercontent.com/1418993/132651629-5458a541-c468-43a1-a4e9-43729cc19e87.png)

I also think this page could do with a warning somewhere about the damage removing entries from the registry can do and that someone should ensure they are comfortable with the risks, At the very least the keys should be exported as a backup :smile:

Reply:
Thank you both - Great points!

Changes have been made but happy to edit further if you'd like :D